### PR TITLE
button for dark light updated

### PR DESCRIPTION
**Pull request by mwhite919**

### CODE REVIEW

File: app/Providers/CryptoProvider.tsx [hunk 1]
```diff
@@ -133,8 +133,9 @@ export default function CryptoProvider({ children }) {
     console.log(palette, e.target.value, "currenttheme");
   }
 
-  function handleMode(e: string) {
-    setMode(e.target.value);
+  function handleMode(value: string) {
+    setMode(value);
+    console.log("mode", mode, value, "value");
```
Comment (yoloOnTheBattlefield) [inline app/Providers/CryptoProvider.tsx]: nop console

File: components/Navigation.tsx [hunk 2]
```diff
@@ -127,78 +128,82 @@ export default function Navigation() {
...
+                  onChange={(e) => handlePalette(e)}
+                  name="palette"
+                  className="mr-5 my-5  drop-shadow-md rounded-sm "
                 >
-                  light
-                </button>
+                  <option>Theme</option>
+                  {palettes?.map((theme) => {
+                    return (
+                      <option key={theme} value={theme}>
```
Comment (yoloOnTheBattlefield) [inline components/Navigation.tsx:183]: forgot to add the key

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: components/Navigation.tsx (hunk 2 above)
Comment (mwhite919) [reply components/Navigation.tsx:183]: can the theme not be the key name?

Comment (yoloOnTheBattlefield) [reply components/Navigation.tsx:183]: oh, yes you;re right, but would there be a possibility for that to happen?

**Review by yoloOnTheBattlefield — APPROVED** [review]

File: components/Navigation.tsx (hunk 2 above)
Comment (mwhite919) [reply components/Navigation.tsx:183]: Maybe we are talking about two different parts. For me, there is a key which is the {theme} on line 183. Can you clarify what you are referring to?

**Pull request merged**